app/src/main/res/layout/hedvig_edit_text.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
+<merge xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:id="@+id/datePickLayout"
+        android:layout_width="match_parent"
+        android:layout_height="@dimen/base_margin_sextuple"
+        android:layout_gravity="bottom"
+        android:background="@drawable/background_pick_date"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent">
+
+        <EditText
+            style="@style/Body.Regular"
+            android:id="@+id/textInput"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:layout_marginStart="@dimen/base_margin_double"
+            android:layout_marginEnd="@dimen/base_margin_double"
+            android:background="@null"
+            android:imeOptions="actionDone"
+            android:inputType="numberDecimal"
+            android:singleLine="true"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent" />
+
+    </androidx.constraintlayout.widget.ConstraintLayout>
+
+    <TextView
+        android:id="@+id/textHint"
+        style="@style/Body"
```

Comment:
Avoid using the base `Body`-style, always specify the variant (probably `Body.Regular` in this case.